File: solutions/julia/pangram/1/pangram.jl
"""
    ispangram(input)

Return `true` if `input` contains every alphabetic character (case insensitive).

"""
function ispangram(input)
    alphabet="abcdefghijklmnopqrstuvwxyz"
    is_pangram=true
    for i in 1:26
        is_pangram=occursin(alphabet[i],lowercase(input)) && is_pangram
    end
    is_pangram
end

File: solutions/go/pangram/1/pangram.go
package pangram
import "strings"

func IsPangram(input string) bool {
	is_pangram := true
	alphabet := "abcdefghijklmnopqrstuvwxyz"

	for _, character := range alphabet {
		is_pangram = strings.Contains(strings.ToLower(input), string(character)) && is_pangram
	}
    return is_pangram
}
